rhoas-cli-service-registry/README.adoc

```diff
@@ -251,7 +263,38 @@ The artifact is an Apache Kafka Avro schema in JSON format.
 [source,shell]
 ----
 $ wget https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
+--2021-09-27 16:17:18--  https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
+Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...
```

Comment:
I think that particular output is very verbose and doesn't give any value to end user. wget is also linux specific command. I think we can replace all of that with one sentence - download example file